src/config.ts

```diff
@@ -6,13 +6,14 @@ import type { TelegramConfig } from './channels/telegram.js';
 import type { TwitterConfig } from './channels/twitter.js';
 import env from './env.js';
 
-const filters = z.union([
+const filters = z.discriminatedUnion('name', [
   z.object({ name: z.literal('DAILY_SWAP_SUMMARY') }),
   z.object({ name: z.literal('WEEKLY_SWAP_SUMMARY') }),
   z.object({ name: z.literal('DAILY_LP_SUMMARY') }),
   z.object({ name: z.literal('WEEKLY_LP_SUMMARY') }),
   z.object({ name: z.literal('DAILY_BOOST_SUMMARY') }),
   z.object({ name: z.literal('WEEKLY_BOOST_SUMMARY') }),
+  z.object({ name: z.literal('SWAP_COMPLETED'), minUsdValue: z.number().optional().default(0) }),
```

Comment:
this means we need to adjust the existing config right?

Reply:
Yeah good point, can prepare that for when we deploy